W/Wigxjpf/bundled/quadmath_stub.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Stub implementation for platforms without quadmath support
+ * This file provides error stubs for float128 functions on platforms
+ * where libquadmath is not available (e.g., macOS, ARM Linux)
```

Comment:
The premise that libquadmath isn't available is false: https://github.com/JuliaMath/Quadmath.jl/blob/21441713447a6ec3aa4ce04da544426372c0e306/src/Quadmath.jl#L25-L42

Reply:
Thanks for mentioning this! I will update my build script accordingly.